hafnian/quantum.py

```diff
@@ -659,3 +659,22 @@ def is_pure_cov(cov, hbar=2, sigdigits=6):
             return True
 
     return False
+
+
+def is_classical_cov(cov, hbar=2, sigdigits=6):

```

Comment:
This applies to the previous PR as well (I think I missed it), but it seems more consistent with our other functions to instead have `tol` as an argument, with value `1e-6`.